models/endoscopic_tool_segmentation/docs/README.md

```diff
@@ -3,6 +3,12 @@ A pre-trained model for the endoscopic tool segmentation task and is trained usi
 
 The [PyTorch model](https://drive.google.com/file/d/19yS3t2oLBiB7wT-qeQ82da95VJs_vzRK/view?usp=share_link) and [torchscript model](https://drive.google.com/file/d/1cDZ3Jr7mhpzdzaFyz8yHNowH8k0T1VZz/view?usp=share_link) are shared in google drive. Details can be found in large_files.yml file. Modify the "bundle_root" parameter specified in configs/train.json and configs/inference.json to reflect where models are downloaded. Expected directory path to place downloaded models is "models/" under "bundle_root".
 
+Either an [ImageNet](https://ieeexplore.ieee.org/document/5206848) pre-trained or a local pre-trained encoder weight could be loaded to the model for training, which is controlled by the `use_imagenet_pretrain` parameter in the `configs/train.json` file. If using local pre-trained weight, please update the `load_path`  parameter of the `CheckpointLoader` in the `configs/train.json` with the local weight path.
```

Comment:
Usually say pretrained weights (not weight)

Reply:
I think this part should be rephrased to be more clear. Same issue for the `pathology_nuclei_segmentation_classification` bundle. I tried to modify that bundle as you can see in: https://github.com/Project-MONAI/model-zoo/pull/363/files#diff-a739ca80d65b7994071d4c6f0fc9a2397f698b958b98f1d314056f1fe260d2acR5
but it may still have some issues and may need some experts to double confirm. cc @KumoLiu

Reply:
Update: we've aligned and @zephyrie will help to review it (may submit a separate PR) thus we don't need to worry too much about it on this PR